src/fasta/fasta.cpp

```diff
@@ -30,108 +30,118 @@
 }
 
 size_t
-ebi::vcf::fasta::FileBasedFasta::count(const std::string &contig) const {
-  return mFastaIndex.count(contig);
+ebi::vcf::fasta::FileBasedFasta::count(const std::string &contig) const
+{
+    return fasta_index.count(contig);
 }
 
 ebi::vcf::fasta::RemoteContig::RemoteContig()
-        : mCurl(new ebi::util::curl::Easy())
+        : curl_easy(new ebi::util::curl::Easy())
 {}
 
 class ebi::vcf::fasta::ContigFromENA {
 public:
-  ContigFromENA(const std::string& contigName) {
-    mContigName = contigName;
-    mFastaOutput.reset(new std::ofstream(mContigName));
+  ContigFromENA(const std::string& contigName)
+  {
+      contig_name = contigName;
+      fasta_output.reset(new std::ofstream(contig_name));
   }
 
-  ~ContigFromENA() {
-    mFastaOutput->close();
-    mFastaInput->close();
-    boost::filesystem::remove(mContigName);
+  ~ContigFromENA()
+  {
+      fasta_output->close();
+      fasta_input->close();
+      boost::filesystem::remove(contig_name);
   }
 
-  size_t size() const {
-    return mLength;
+  size_t size() const
+  {
+      return contig_length;
   }
 
-  void write(const char* buffer, const size_t length) {
-    mFastaOutput->write(buffer, length);
-    mLength += length;
+  void write(const char* buffer, const size_t length)
+  {
+      fasta_output->write(buffer, length);
+      contig_length += length;
   }
 
-  std::string read(const size_t pos, const size_t length) {
-    if ( !mFastaInput.get() ) {
-      mFastaInput.reset(new std::ifstream(mContigName, std::ios_base::in));
-    }
+  std::string read(const size_t pos, const size_t length)
+  {
+      if (!fasta_input.get()) {
+          fasta_input.reset(new std::ifstream(contig_name, std::ios_base::in));
+      }
 
-    std::string result;
-    return ebi::util::read_n(*mFastaInput, result, length, pos);
+      std::string result;
+      return ebi::util::read_n(*fasta_input, result, length, pos);
   }
 
 private:
-  size_t mLength;
-  std::string mContigName;
-  std::unique_ptr<std::ifstream> mFastaInput;
-  std::unique_ptr<std::ofstream> mFastaOutput;
+  size_t contig_length;
+  std::string contig_name;
+  std::unique_ptr<std::ifstream> fasta_input;
+  std::unique_ptr<std::ofstream> fasta_output;
 };
 
 std::string
-ebi::vcf::fasta::RemoteContig::sequence(const std::string& contig, const size_t pos, const size_t length) {
-  if ( mContig.count(contig) == 0 ) {
-    mContig[contig].reset(new ebi::vcf::fasta::ContigFromENA(contig));
+ebi::vcf::fasta::RemoteContig::sequence(const std::string& contig, const size_t pos, const size_t length)
+{
+    if (contigs.count(contig) == 0) {
+        contigs[contig].reset(new ebi::vcf::fasta::ContigFromENA(contig));
 
-    // This contig is not downloaded, try download it from ENA.
-    if (contig.size() < 6) { // don't trust short contig names.
-      return "";
-    }
+        // This contig is not downloaded, try download it from ENA.
+        if (contig.size() < 6) { // don't trust short contig names.
```

Comment:
The intention was not to download the referene sequence for a chromosome like `chr22`, but this will prevent downloading for contigs with short accessions.
After a further look, this is not absolutely necessary as we will get an empty reference sequence if the short name is not supported. I have hence removed this check in dc4dda3 .